packages/xo-web/src/xo-app/hub/recipes/recipe.js

```diff
@@ -51,15 +51,38 @@ export default decorate([
           size: 'medium',
         })
 
-        const { masterName, nbNodes, network, sr, sshKey } = recipeParams
+        const {
+          gatewayIpAddress,
+          masterIpAddress,
+          masterName,
+          nbNodes,
+          network,
+          networkMask,
+          sr,
+          sshKey,
+          staticIpAddress,
+        } = recipeParams
+
+        let workerNodeIpAddresses
+        if (staticIpAddress === true) {
```

Comment:
```suggestion
        if (staticIpAddress) {
```

Reply:
Maybe I have misunderstood something but Julien told me in a previous PR that it's better to use explicit comparaison.

https://github.com/vatesfr/xen-orchestra/pull/6534#discussion_r1031633834